src/main/msp/msp.c

```diff
@@ -4185,6 +4185,13 @@ static mspResult_e mspCommonProcessInCommand(mspDescriptor_t srcDesc, int16_t cm
                     // increase the canvas size to the HD default as that is what the user will expect
```

Comment:
Maybe insert
`#ifdef USE_OSD_HD`
here, because we should only make these changes if the build supports HD